pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -67,6 +67,8 @@ var (
 		ShutdownGracePeriod:             metav1.Duration{Duration: 30 * time.Second},
 		ShutdownGracePeriodCriticalPods: metav1.Duration{Duration: 10 * time.Second},
 		MemoryThrottlingFactor:          utilpointer.Float64(0.9),
+		ReconcilerLoopSleepPeriod:       metav1.Duration{Duration: 100 * time.Second},
```

Comment:
is `100 * time.Millisecond` or `100 * time.Second`

Reply:
Hello @googs1025 ,
This test is intended solely to validate the correct configuration, not to verify the default configuration when the field is not set. 
one of the default value test : pkg/kubelet/apis/config/scheme/testdata/KubeletConfiguration/after/v1beta1.yaml  ( with empty conf on before  yaml file )

Reply:
But is 100 * time.Second the correct configuration?

Reply:
Yep 
(Batch interval with 100 seconds : user needs  )
The impact « reasonable «  : instead of iterating with 100ms -> 100 seconds : longer reconciliation period ( user needs )